Assets/Scripts/UI/Pickup Pup/DogSlot/DogSlot.cs

```diff
@@ -8,201 +8,201 @@
 
 public class DogSlot : PPUIElement
 {
-	protected bool hasDogInfo
-	{
-		get
-		{
-			return dogInfo != null;
-		}
-	}
-
-	protected bool hasDog
-	{
-		get
-		{
-			return dog != null;
-		}
-	}
-
-	protected Dog dog
-	{
-		get
-		{
-			return _dog;
-		}
-		set
-		{
-			// Fixes ref on previous dog
-			if(_dog != null)
-			{
-				_dog.LeaveCurrentSlot();
-			}
-			// Assigns slot to new dog (assuming the new value is not null)
-			if(value != null)
-			{
-				value.AssignSlot(this);
-			}
-			_dog = value;
-		}
-	}
-
-	bool hasButton 
-	{
-		get 
-		{
-			return button != null;
-		}
-	}
-		
+    protected bool hasDogInfo
```

Comment:
Not sure what's going on with this class, but generally advise avoiding adding empty changes like this. Doesn't seem to change the code at all, and could easily lead to unnecessary merge conflicts